indico/web/templates/forms/location_widget.html

```diff
@@ -1,34 +1,66 @@
+{# The location widget provides 3 core fields to describe the location; venue,
```

Comment:
I could put it as a docstring, but I thought it makes more sense here, cause what is described in the comment can already be seen in the code below.